src/components/GGanttCurrentTime.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div
+    class="g-grid-current-time"
+    :style="{
+      left: `${xDist}px`
+    }"
+  >
+    <div
+      class="g-grid-current-time-marker"
+      :style="{
+        border: `1px dashed ${colors.markerCurrentTime}`
+      }"
+    />
+    <span class="g-grid-current-time-text" :style="{ color: colors.markerCurrentTime }">TODAY</span>
```

Comment:
"TODAY" is not always a fitting label, e.g. in case the time range of the chart (chart-start - chart-end) is within a single day. It'd be good to omit the label or, even better, make it an optional prop.

Reply:
by default, i changed the span to `NOW`, but now it's possible to change the `span` and the `div` with a `<slot>` tag

Default behaviour:
![image](https://github.com/zunnzunn/vue-ganttastic/assets/51349076/88be3e2f-696b-4aec-aa79-4796c0bd40ba)


Custom behaviour:
![image](https://github.com/zunnzunn/vue-ganttastic/assets/51349076/12875911-2f19-4c10-bed0-95391789a520)
![image](https://github.com/zunnzunn/vue-ganttastic/assets/51349076/f619aa1d-ebb7-4165-9572-2326fd744b67)